.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: RELEASE
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  test:
+    name: Release ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
```

Comment:
Let's not use `matrix` in that case (we're publishing in context of one env)

Reply:
Done